src/components/dev-dot-content/mdx-components/mdx-hcp-callout/index.tsx

```diff
@@ -75,8 +75,8 @@ export default function MdxHcpCallout({ product }: MdxHcpCalloutProps) {
 					color="var(--white)"
 					className={s.subHeading}
 				>
-					To read core {productName} documentation and tutorials, including
-					local open source docs
+					Read Core {productName} documentation and tutorials, including
```

Comment:
```suggestion
					Read core {productName} documentation and tutorials, including
```

Reply:
I know that's what was suggested but I have a feeling we want to keep the right casing lol